ExtensionStats/ExtensionStats.py

```diff
@@ -338,7 +338,8 @@ def __init__(self):
       '4.6.0': '25441',
       '4.6.2': '25516',
       '4.8.0': '26489',
-      '4.8.1': '26813'
+      '4.8.1': '26813',
+      '4.9.0': '26493'
```

Comment:
Should the revision of the nightly be retrieved dynamically ? Using something like [this](https://stackoverflow.com/a/623396/1539918)

Reply:
I see. It may be sensible to update the documentation of [this function](https://github.com/fbudin69500/SlicerDeveloperToolsForExtensions/blob/5565ece41625f6ec3998bbdba41c58e029a9e74f/ExtensionStats/ExtensionStats.py#L399-L401).

From 

```
      Return Slicer release name that corresponds to a Slicer revision.
      Downloads associated with nightly build happening between release A and B are
      associated with A-nightly "release".
```

to

```
      Return Slicer release name that corresponds to a Slicer revision.
      Downloads associated with nightly build happening between release A and B are
      associated with A-nightly "release".
      Downloads associated with nightly build happening after the most recent release C
      are also associated with C-nightly "release".
```

> '4.9.0': '26493'

It may be worth updating https://www.slicer.org/wiki/Release_Details indicating the revision associated with 4.9.0 and mentioning that it corresponds to the development release.